AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Controllers/SiteAdminControllerShould.cs

```diff
@@ -136,9 +136,24 @@ public void EditUserGetSendsUserByUserIdQueryWithCorrectUserId()
             mediator.Verify(m => m.Send(It.Is<UserByUserIdQuery>(q => q.UserId == userId)), Times.Once);
         }
 
-        [Fact(Skip = "NotImplemented")]
+        [Fact]
         public void EditUserGetReturnsCorrectViewModelWhenOrganizationIdIsNull()
         {
+            {
+                var mediator = new Mock<IMediator>();
+                var logger = new Mock<ILogger<SiteController>>();
```

Comment:
@SteveKennedy you should be able to pass in null for the `ILogger<SiteController> logger` dependency b/c the EditUser action method does not invoke any members of ``ILogger<SiteController>`. Besides that, looks good.